cmd/system-probe/check_cmd.go

```diff
@@ -35,7 +35,7 @@ func querySocketEndpoint(cfg *config.AgentConfig, check string, client string) e
 			MaxIdleConns:    2,
 			IdleConnTimeout: 30 * time.Second,
 			DialContext: func(_ context.Context, _, _ string) (net.Conn, error) {
-				return net.Dial("unix", cfg.SystemProbeSocketPath)
+				return net.Dial("unix", cfg.SystemProbeAddress)
```

Comment:
will this get called in windows? And doesn't the "unix" argument cause problems?

Reply:
Hey, so I think this will ultimately be removed by pedro's changes here (the entire check_cmd file)

- https://github.com/DataDog/datadog-agent/pull/5544/files

so would it be okay to leave it for now?

Reply:
Spoke over dm -> is okay to leave it as-is for now.